purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -2278,3 +2307,22 @@ internal fun Purchases.AttributionNetwork.convert(): CommonAttributionNetwork {
     return CommonAttributionNetwork.values()
         .first { attributionNetwork -> attributionNetwork.serverValue == this.serverValue }
 }
+
+private fun checkMethodAvailability(
```

Comment:
i don't love this, but not expecting it to stay around as we think of a longer-term solution

Reply:
maybe this is overkill now that it's only being used for one method..unless we think we'll use it again in the future?

Reply:
Yes, I think I agree this is a bit overkill. I think we can just check directly if the store is Amazon, then log in the canMakePayments function.